src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.cs

```diff
@@ -751,5 +752,52 @@ public async Task When_TouchTap_Then_NestedElementReceivePointerEvents()
 
 			events.Should().BeEquivalentTo("enter", "pressed", "release", "exited");
 		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+#if !__SKIA__
+		[Ignore("Pointer injection supported only on skia for now.")]
+#endif
+		public async Task When_ReversedMouseWheel_Then_ScrollInReversedDirection()
+		{
+#if WINDOWS_UWP
+			Assert.Inconclusive("Mouse pointer helper not supported on UWP.");
```

Comment:
Consider adding `await Task.CompletedTask;` for WINDOWS_UWP to avoid disabling CS1998

Reply:
As far as I know the CS1998 is irrelevant in (runtime) tests (and actually in most code as it's almost only to ensure to not add overhead of the async/await state machine when not needed). The relevant one is CS4014 as it will ensure that a `Task` returned by a method is being awaited.

Adding `await Task.CompletedTask;` is only a trick to hide the rule without adding a proper ignore, and I suspect it will even be less performant (as it will generate the state machine for the `CompletedTask` it would not with the ignore). It also unnecessarily complexity the source code.

Reply:
Technically, they are functionally different. While the difference isn't going to be noticeable for this specific case, it might be for other cases.

```csharp
using System;
using System.Threading.Tasks;

_ = new C().M();
Console.WriteLine("Exception is swallowed"); // if M is async, this is printed. Otherwise, the exception is thrown. This happens only when the returned task isn't awaited.

public class C
{
    public
        async
        Task<C> M()
    {
        throw new Exception();
    }
}
```

Reply:
Oh, my suggestion to add `await Task.CompletedTask;` will actually keep the `async` which will still swallow expcetions for non-awaited returned tasks.

Reply:
No strong opinion given this is a test code, but I'd personally add `async` modifier conditionally (rather than `await Task.CompletedTask;` which is pointless), or at least suppress for this specific case with `#pragma warning disable`.

My concern with globally disabling the warning for runtime tests is cases where exceptions are unexpectedly swallowed (but I also see it not being a common mistake we could do)

Reply:
Yes the case `_ = new C().M();` is why I said "almost", but here the difference in behavior is mainly because you also added the "ignore CS4014 trick" by adding the `_ =`, if you do the standard `await new C().M();` the exception is being raised properly.

Adding conditionally the `async` is also another option, but again adds noise in the source code. Another option would also to just put the whole test in the 1 #if !WINDOWS_UWP` but this as the disadvantage to hide tests that are not running on a specific platform.

As it's for **multi-targeted** tests, where it's quite common to have platform specific code like this, I would rather rely on the CS4014 than on the CS1998 and ease usage with a global ignore like this (in this project only of course!). Any opinion @jeromelaban ?